### DISCUSSION TITLE
[General]: Emacs.app alias file at /Applications folder

### DISCUSSION THREAD

**Discussion by BoundaryValueProblems** [d12frosted/homebrew-emacs-plus#659] [General]
### Issue description

Via homebrew, my emacs-plus system was updated to 29.2. However, unlike the previous version, it was not copied or symbolically linked to /Applications/Emacs.app correctly. Instead, it suggested to run the following:
`osascript -e 'tell application "Finder" to make alias file to posix file "/usr/local/opt/emacs-plus@29/Emacs.app" at POSIX file "/Applications"'`
It poped up the window, and I pushed "Allow". It is not clear to me what it did, but it created a file /Applications/Emacs.app alias. And /Applications/Emacs.app was linked to the previous version 29.1. I am wondering why this time the installation process is different from the previous one. In the end I had to remove the link and relink to the correct folder manually. 
Can I remove /Applications/Emacs.app alias? (I guess so, but I want to know what this file is).
Thanks for your help!

**Comment by d12frosted** [comment]
Yes, feel free to remove it.
